api/src/prescription/learner-management/infrastructure/serializers/csv/csv-organization-learner-parser.js

```diff
@@ -119,22 +143,24 @@ class CsvOrganizationLearnerParser {
     return learnerAttributes;
   }
 
-  _checkColumns(parsedColumns) {
+  _checkColumns(parsedColumns, errors) {
```

Comment:
**question** pourquoi on utilise un param `errors` alors qu'on pourrait utiliser  `this._errrors`